locales/fr.json

```diff
@@ -49,6 +53,10 @@
       "description": "S’assure que les éléments <audio> ont des sous-titres",
       "help": "Les éléments <audio> doivent avoir une psite de sous-titres"
     },
+    "autocomplete-valid": {
+      "description": "S’assure que l’attribut autocomplete est correct et adapté pour le champ de formulaire",
+      "help": "Lgattribut autocomplete doit être utilisé correctement"
```

Comment:
Typo on the first word (should be `L'attribut`)

Reply:
`L’attribut` even :-)